packages/chart/src/data/initial-state.js

```diff
@@ -68,9 +68,6 @@ export default {
   boxplot: {
     plots: [],
     borders: 'true',
-    firstQuartilePercentage: 25,
```

Comment:
It looks like we can remove these from ts as well

Reply:
removed